src/bindings/python/flux/job/Jobspec.py

```diff
@@ -1001,7 +1099,21 @@ def from_command(
         resources = [resource_section]
         tasks = [{"command": command, "slot": "task", "count": task_count_dict}]
         attributes = {"system": {"duration": 0}}
-        return cls(resources, tasks, attributes=attributes)
+        return cls(resources, tasks, attributes=attributes)._set_extra_options(
+            duration=duration,
+            environment=environment,
+            env_expand=env_expand,
+            cwd=cwd,
+            rlimits=rlimits,
+            name=name,
```

Comment:
Why `name` and not `jobname` or `job_name`?

Reply:
It seems more consistent: the other args are not `job_cwd`, `job_rlimits`, `job_environment` and so on, so why tack `job_` in front of the name?

Reply:
In the case of (1) and (3) the command line has `--cwd` and `--env` while the command line option to name a job is `--job-name`. My first instinct would have been to try and match the command line, but I don't think it really matters all that much.